### PR TITLE
Fix for Jenkins failure on JMSTestCase (#894)

### PR DESCRIPTION
**Pull request by grainier**
Fix for Jenkins failure on JMSTestCase [#894] (https://wso2.org/jenkins/job/product-cep/org.wso2.cep$org.wso2.cep.integration.test/894/)

### CODE REVIEW

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java [hunk 1]
```diff
@@ -52,34 +46,34 @@
 
 public class Wso2EventServer implements Runnable {
     private static Log log = LogFactory.getLog(Wso2EventServer.class);
+    private final String FILE_STREAM_DEFINITION_EXT = ".json";
+    AbstractStreamDefinitionStore streamDefinitionStore = new InMemoryStreamDefinitionStore();
```
Comment (mohanvive) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: Make this attribute as private..

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java [hunk 2]
```diff
@@ -153,9 +148,9 @@ public void run() {
         try {
             startServer();
         } catch (DataBridgeException e) {
-            e.printStackTrace();
+            log.error(e.getMessage(), e);
```
Comment (mohanvive) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: This is not a correct way of logging.  Add a proper log message and stack-trace. Check other places as well.

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java [hunk 3]
```diff
@@ -184,16 +179,14 @@ public long getMsgCount() {
...
 
-        if(listStreamDefinitionFiles != null){
+        if (listStreamDefinitionFiles != null) {
             for (final String fileEntry : listStreamDefinitionFiles) {
-
                 BufferedReader bufferedReader = null;
                 StringBuilder stringBuilder = new StringBuilder();
-                String fullPathToStreamDefinitionFile = directoryPath + "/"+ fileEntry;
-
+                String fullPathToStreamDefinitionFile = directoryPath + "/" + fileEntry;
```
Comment (ramindu90) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: Replace "/" with File.separator

Comment (grainier) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: Fixed in #196

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java (hunk 2 above)
Comment (grainier) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: Fixed in #196

File: modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java (hunk 1 above)
Comment (grainier) [inline modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java]: @mohanvive Fixed in #196

**Pull request merged**